### ISSUE TITLE
Not accepting new config file version

### ISSUE THREAD

**Issue by Kitcat490** [dvehrs/podget#12]
When creating a new config file with podget 0.8, it complains that the config needs to be created by a version newer than 0.7.0, giving the following error:

"kitcat490@r52 ~ $ podget
podget
/home/kitcat490/.podget/podgetrc was created by an older version of podget than is needed by this version.

This version of podget requires a configuration produced for version 0.7.0 or newer.  This configuration
was produced by version 0.8

Please update your configuration by creating a new one with the command 'podget --create-config <FILENAME>'
and then comparing its contents to your old configuration.  Once the new file has been updated to reflect
your desired configuration, move it in place of the old one.

Closing session."

**Comment by dvehrs** [comment]
OK, good catch.

This happened because I had created a way of extracting the parts of the version number that depended on there being three parts.  So '0.8.0' should have worked, but it just didn't seem that elegant so I used '0.8'.

I've updated the version number parsing use an array and assign the value of '0' to any undefined parts.  It now assumes the first part is the BASE, the second is the MAJOR and the third (if defined) is the MINOR.  This should also allow us to define further version subparts beyond MINOR as needed and not cause podget to fail this check.  

I've uploaded my proposed fix to the 'dev' branch and with a little more testing will push it to 'master' with a new version release.

Thanks,

Dave
